src/viam/utils.py

```diff
@@ -258,8 +258,8 @@ async def to_thread(func: Callable[_P, _R], *args: _P.args, **kwargs: _P.kwargs)
 
 
 def from_dm_from_extra(extra: Optional[Dict[str, Any]]) -> bool:
-    """Used in modular filter components to get the 'fromDataManagement' value from an extra map."""
+    """Used in modular filter components to return whether 'fromDataManagement' was set in the extra map."""
     if extra is None:
         return False
 
-    return bool(extra["fromDataManagement"])
+    return "fromDataManagement" in extra
```

Comment:
isn't it possible that "fromDataManagement" is in extra but not true?

Reply:
The only time we pass "fromDM" is when it's explicitly set to true - but changed this to return the value.

Reply:
that sounds good, returning value is a bit more robust so thanks for making the change!